crates/nargo/src/artifacts/debug.rs

```diff
@@ -0,0 +1,47 @@
+use serde::{Deserialize, Serialize};
+use std::collections::{BTreeMap, BTreeSet};
+
+use fm::FileId;
+use noirc_errors::debug_info::DebugInfo;
+use noirc_frontend::hir::Context;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DebugFile {
+    pub source: String,
+    pub path: String,
```

Comment:
Thinking out loud: should this be a String or perhaps a Path?

Reply:
I think Path cannot be Deserialize or stored in a BtreeMap because it doesn't implement Sized

Reply:
Oh, I think we can use PathBuf

Reply:
yeah, we should use pathbuf